package.json

```diff
@@ -16,6 +16,7 @@
     "start": "node server.js",
     "clean": "rimraf dist",
     "build": "npm run clean && npm run build:webpack",
+    "build:dev": "webpack --config webpack.config.js",
```

Comment:
I tried parallel build, provided by happypack, only around 10secs improvement, deduplication and minifying are taking half the time

And there's some analyse tool from webpack site which can tell you what to optimize, I've yet understand though

Right now this is the cheapest way to gain the speed